schemas/Manual.regions.schema.json

```diff
@@ -29,14 +29,15 @@
                             "description": "Array of other regions the player can go from this region.",
                             "type": "array",
                             "items": {
-                                "$ref": "#/definitions/Connects_to"
+                                "type": "string"
                             },
                             "minItems": 0,
                             "uniqueItems": true
                         },
                         "starting": {
                             "description": "(Optional) Is this region accessible from the start? Defaults to false if not included.",
```

Comment:
Again, do we need the default in the description here?

Reply:
All defaults should be gone from the descriptions 😄